flowredux/src/commonMain/kotlin/com/freeletics/flowredux/dsl/ChangeState.kt

```diff
@@ -21,12 +21,17 @@ public sealed class ChangeState<out S>
 /**
  * Sets a new state by directly override any previous state
  */
+@Deprecated("call override() on State instead", level = DeprecationLevel.WARNING)
 public data class OverrideState<S>(internal val newState: S) : ChangeState<S>()
```

Comment:
Kept these public as deprecated for now to make the migration at least a bit easier

Reply:
Also in general